docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss

```diff
@@ -22,8 +22,8 @@
   }
 }
 
-// Decanter classes used in Content Type WYSIWYGs and in Text Area WYSIWYG components
-.decanter {
```

Comment:
should I keep this as a fallback?

Reply:
My instinct is no. If we keep it, then there could be a mix of use cases. One person might continue to use `decanter` while another adopts the use of `hs`. Removing it forces everyone to use `hs` and creates consistency moving forward.

Although I would touch base with Bryan to confirm just in case there is a use case that I'm not considering.

Reply:
I agree, let's remove the fallback. We will need to mention this to H&S... things are going to look broken on our test site for a bit, but it's what's best long-term.